tests/queries/0_stateless/01753_system_zookeeper_query_param_path.reference

```diff
@@ -0,0 +1 @@
+clickhouse
```

Comment:
Test will fail if `/` has other children. Try to execute something like 
```
CREATE TABLE rmt (n Int8) ENGINE=ReplicatedMergeTree('/other/path/rmt', 'r')
DROP TABLE rmt SYNC
```
and run this test.
See also `01700_system_zookeeper_path_in`